.github/workflows/avm.platform.set-avm-github-issue-owner-config.yml

```diff
@@ -22,7 +22,8 @@ jobs:
         shell: pwsh
         run: |
           # Load used functions
-          . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines'  'platform' 'Set-AvmGitHubIssueOwnerConfig.ps1')
+          . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines' 'platform' 'Get-AvmCsvData.ps1')
```

Comment:
Function does not appear to be used

Reply:
This may be - but it would be impossible to tell from the pipeline and even harder to understand later ;) Loading a functino that is used by a script should happen in that script and not outside. ;)

Reply:
Added in other comment.

Please note, you should then also pass the repo root in explicitely (just in case):

```PowerShell
$functionInput = @{
  Repo     = "${{ github.repository_owner }}/${{ github.event.repository.name }}"
  RepoRoot = $env:GITHUB_WORKSPACE
}
```